language/de/common.php

```diff
@@ -452,6 +456,7 @@
 	'NOTIFICATION_FORUM'				=> '<em>Forum:</em> %1$s',
 	'NOTIFICATION_GROUP_REQUEST'		=> '<strong>Beitrittsgesuch</strong> von %1$s für Gruppe %2$s.',
 	'NOTIFICATION_GROUP_REQUEST_APPROVED'	=> '<strong>Beitrittsgesuch akzeptiert</strong> für Gruppe %1$s.',
+	'NOTIFICATION_METHOD_INVALID'		=> 'Die Methode „%s“ verweist nicht auf eine gültige Benachrichtigungs-Methode.',
```

Comment:
Das finde ich etwas sperrig. Vielleicht:
> Die Methode "%s" bezeichnet keine gültige Benachrichtigungs-Methode

oder 
> Die Methode "%s" ist keine gültige Benachrichtigungs-Methode

Ich nehme mal an, dass es um die in der DB abgelegten Klassenbezeichnungen der notifications geht, anhand der die eigentliche Klasse bestimmt wird, also z.B. \phpbbde\whwz\notification\method\reminder